### PR TITLE
Start to split up Omicron README into easier-to-navigate pieces

### PR DESCRIPTION
**Pull request by smklein**

### CODE REVIEW

File: docs/cli.adoc [hunk 1]
```diff
@@ -0,0 +1,182 @@
+:showtitle:
+:toc: left
+:icons: font
+
+= Oxide CLI
+
+The `oxide` CLI is used to access Nexus' external API, which is
+the public interface to Omicron.
+
+For more detail, refer to https://docs.oxide.computer/cli/manual[`oxide`'s CLI Manual].
```
Comment (jordanhendricks) [inline docs/cli.adoc:10]: In the rendered version (https://github.com/oxidecomputer/omicron/blob/c6ec4d0c919619d4c2bf133b61385ce096b2793f/docs/cli.adoc#oxide-cli), I see a stray backtick. I think the backticks around "oxide" on this line can be removed

File: docs/cli.adoc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  "diskId": "551bbe67-3640-41c9-b968-249a136e5e31",
+  "diskName": "nginx",
+  "diskState": {
+    "state": "attached",
+    "instance": "99ad2514-050c-4493-9cb9-d9ceba980a98"
+  }
+}
+
+# Alternatively, you can use the API command to run any endpoint.
+# This operates like a fancy, authenticated curl.
```
Comment (jordanhendricks) [inline docs/cli.adoc:108]: Could this be pulled out of the code block? It gets a little hard to read among the help message text below.

**Review by jordanhendricks — APPROVED** [review]
Hurray! This is a much clearer way of organizing the information in the README. I had a couple of small changes requested for readability but otherwise this looks great!

File: docs/cli.adoc (hunk 1 above)
Comment (smklein) [reply docs/cli.adoc]: Removed

File: docs/cli.adoc (hunk 2 above)
Comment (smklein) [reply docs/cli.adoc]: Sure thing, done

File: docs/cli.adoc [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+$ oxide project create myproject \
+    --description "My project" \
+    --organization myorg
+✔ Created project myorg/myproject
+
+$ oxide instance create myinstance \
+    --description "My instance" \
+    --project myproject \
+    --org myorg \
```
[the remark this answers is not in the document]
Comment (smklein) [reply docs/cli.adoc]: updated

**Pull request merged**